pkg/env/env.go

```diff
@@ -243,6 +243,7 @@ func (e *testEnv) processTests(t *testing.T, enableParallelRun bool, testFeature
 
 	var wg sync.WaitGroup
 	for i, feature := range testFeatures {
+		feature := feature
```

Comment:
I would suggest couple of things:
1. For clarity, name the new variable something else other than `feature`
2. Have your anonymous function (line 253) take a feature, as well, as parameter

```go
if runInParallel {
			wg.Add(1)
			go func(w *sync.WaitGroup, f Feature) {
				defer w.Done()
				e.processTestFeature(t, f.Name, f)
			}(&wg, feature)
		} else {
			e.processTestFeature(t, featName, feature)
		}
```

Reply:
Thanks for the review @vladimirvivien . Addressed the requests.